data/Raj210Kumari.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Rajnandani Kumari",
+  "type": "personal",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
+  "links": [
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/Raj210Kumari",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Checkout my portfolio",
+      "url": "http://raj210kumari.github.io/",
+      "icon": "FaWeb"
```

Comment:
`FaWeb` is not a valid icon. You can find any relevant icon from here.
https://linkfree.eddiehub.io/icons